src/commands/utils/roleAssignments.ts

```diff
@@ -26,6 +26,11 @@ export function getScopeForAcr(subscriptionId: string, resourceGroup: string, ac
     return `/subscriptions/${subscriptionId}/resourceGroups/${resourceGroup}/providers/${acrProvider}/registries/${acrName}`;
 }
 
+export function getScopeForCluster(subscriptionId: string, resourceGroup: string, clusterName: string): string {
+    // ARM resource ID for AKS
```

Comment:
💡 MInor: might be for the patter below add a sample in the comment or doc link for reference.